controller/workitem.go

```diff
@@ -256,10 +256,21 @@ func (c *WorkitemController) Create(ctx *app.CreateWorkitemContext) error {
 				return jsonapi.JSONErrorResponse(ctx, errors.NewBadParameterError("space", "string").Expected("valid space ID"))
 			}
 		}
+
+		if _, ok := wi.Fields[workitem.SystemArea]; ok == false {
+			// no area assigned yet hence set root area
+			rootArea, err := appl.Areas().Root(ctx, spaceID)
+			if err != nil {
+				return jsonapi.JSONErrorResponse(ctx, errs.Wrap(err, fmt.Sprintf("Error fetching Root Area")))
```

Comment:
not capitalized error messages https://github.com/golang/go/wiki/CodeReviewComments `Error String` section